package.json

```diff
@@ -63,29 +63,30 @@
   "dependencies": {
     "@angular/animations": "6.0.0-rc.1",
     "@angular/bazel": "6.0.0-rc.1",
-    "@angular/cdk": "^5.2.0",
-    "@angular/cli": "^1.7.0",
+    "@angular/cdk": "^6.0.0-rc.0",
+    "@angular/cli": "6.0.0-rc.0",
     "@angular/common": "6.0.0-rc.1",
     "@angular/compiler": "6.0.0-rc.1",
     "@angular/compiler-cli": "6.0.0-rc.1",
     "@angular/core": "6.0.0-rc.1",
     "@angular/forms": "6.0.0-rc.1",
     "@angular/http": "6.0.0-rc.1",
-    "@angular/material": "^5.2.0",
+    "@angular/material": "^6.0.0-rc.0",
     "@angular/platform-browser": "6.0.0-rc.1",
     "@angular/platform-browser-dynamic": "6.0.0-rc.1",
     "@angular/platform-server": "6.0.0-rc.1",
     "@angular/router": "6.0.0-rc.1",
-    "@ngrx/db": "github:brandonroberts/ngrx-db-builds",
+    "@ngrx/db": "^2.2.0-beta.0",
     "core-js": "^2.5.4",
     "hammerjs": "^2.0.8",
     "ngrx-store-freeze": "^0.2.1",
     "opencollective": "^1.0.3",
-    "rxjs": "6.0.0-beta.4",
-    "rxjs-compat": "6.0.0-beta.1",
+    "rxjs": "6.0.0-rc.0",
+    "rxjs-compat": "6.0.0-rc.0",
```

Comment:
Is this still needed?

Reply:
For now, yes, but only for `husky`. We are using `lint-staged` which has a dependency on `listr` and its still using RxJS 5.x imports.

https://github.com/SamVerschueren/listr/blob/master/lib/task.js#L4